.github/workflows/test.yaml

```diff
@@ -7,12 +7,12 @@ jobs:
     - name: Setup go
       uses: actions/setup-go@v2
       with:
-        go-version: '^1.17'
+        go-version: '^1.18'
     - name: Checkout repository
       uses: actions/checkout@v2
     - name: golangci-lint
       uses: golangci/golangci-lint-action@v2
       with:
-        version: v1.43
+        version: v1.45
```

Comment:
Not for this PR, but we should probably update the Makefile in `goks` to be closer to `koko`. What I mean is we should update the `lint` to look like [this](https://github.com/Kong/koko/blob/b9bc99152ba86af9f3e83e1c00b9b7101347c549/Makefile#L20) and then we can create a directory for scripts and include a version of `install_tools.sh` that just [installs golangci-lint](https://github.com/Kong/koko/blob/main/scripts/install-tools.sh#L26-L29) and update this Makefile to add a `install-tools` [action](https://github.com/Kong/koko/blob/b9bc99152ba86af9f3e83e1c00b9b7101347c549/Makefile#L4-L6)

Reply:
yeah good point, I'll file a ticket to track this